08-projects.Rmd

```diff
@@ -6607,39 +6607,37 @@ F.R.I.E.N.D.S is a popular tv show. They have an interesting naming convention f
 - First 5 results of the query.
 ```
 
-The next couple of questions should be complete in the same language. You can use either R or Python, but you must use the same for both questions.
```

Comment:
This paragraph is removed assuming Python is not needed in this project.

Reply:
Agreed, thank you!